emcc.py

```diff
@@ -1222,7 +1245,7 @@ def get_bitcode_args(input_files):
     # First, generate LLVM bitcode. For each input file, we get base.o with bitcode
     for i, input_file in input_files:
       file_ending = filename_type_ending(input_file)
-      if file_ending.endswith(SOURCE_ENDINGS):
+      if file_ending.endswith(SOURCE_ENDINGS) or has_fixed_language_mode:
```

Comment:
Oh sorry. Yes, I understand now :) Ignore my previous comment. I will do a change in a moment.